test/gemm/gemm_test.hpp

```diff
@@ -91,7 +95,26 @@ namespace rocwmma
 
             kernel->exec();
             kernel->validateResults();
-            kernel->reportResults();
+
+            if (!loggingOptions->omitCout())
+            {
+                kernel->reportResults(std::cout, 
+                                    false,
```

Comment:
Pass in the static header boolean here to either print the header or not. Repeat this for all logger streams.